include/constants/pokemon.h

```diff
@@ -312,9 +312,9 @@
 #define FLAG_POWDER                 (1 << 19)
 #define FLAG_TARGET_ABILITY_IGNORED (1 << 20)
 #define FLAG_DANCE                  (1 << 21)
-#define FLAG_DMG_IN_AIR             (1 << 22) // X2 dmg on air, always hits target on air
-#define FLAG_HIT_IN_AIR             (1 << 23) // dmg is normal, always hits target on air
-#define FLAG_DAMAGE_AIRBORNE        (1 << 24) // Makes a Ground type move do 1x damage to flying and levitating targets
+#define FLAG_DMG_2X_IN_AIR          (1 << 22) // If target is in the air, can hit and deal double damage.
+#define FLAG_DMG_IN_AIR             (1 << 23) // If target is in the air, can hit.
+#define FLAG_DMG_UNGROUNDED_IGNORE_TYPE_IF_FLYING (1 << 24) // Makes a Ground type move do 1x damage to flying and levitating targets
```

Comment:
Note on this name: it looks completely stupid but it's meant to be a temporary name since refactoring thousand arrows is out of the scope of this pr